lib/hanami/action.rb

```diff
@@ -58,8 +58,7 @@ def self.gem_loader
 
     # See {Config} for individual setting accessor API docs
     setting :handled_exceptions, default: {}
-    setting :format_mappings, default: Config::DEFAULT_FORMAT_MAPPINGS
```

Comment:
@timriley The main point of this commit is to have only `config.formats` instead of two: `format_mappings` and `formats`.